app/services/return_book.rb

```diff
@@ -0,0 +1,21 @@
+class ReturnBook
+  def self.process(book_id:, user_id:)
+    new(book_id: book_id, user_id: user_id).process
+  end
+
+  def initialize(book_id:, user_id:)
+    @book_id = book_id
+    @user_id = user_id
+  end
+
+  def process
+    checkout = Checkout.where(book_id: @book_id, user_id: @user_id).last
```

Comment:
I would suggest adding an explicit order to this so you don't accidentally call `last` and you get the wrong one